Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,265 @@
+            style: style,
+            action: {
+                value -= 1
+            }
+        )
+        .accessibility(.stepperActionDecrement)
+    }
+    
+    @ViewBuilder var incrementButton: some View {
+        StepperButton(
+            isIncrement: true,
+            isEnabled: isEnabled ? value < maxValue : false,
+            style: style,
+            action: {
+                value += 1
+            }
+        )
+        .accessibility(.stepperActionIncrement)
+    }
+    
+    /// Creates Orbit Stepper component.
+    public init(
+        value: Binding<Int>,
+        minValue: Int,
+        maxValue: Int,
+        style: Style
+    ) {
+        self._value = value
+        self.minValue = minValue
+        self.maxValue = maxValue
+        self.style = style
+    }
+}
+
+// MARK: - Subtypes
+extension Stepper {
+    
+    struct StepperButton: View {
+        
+        let isIncrement: Bool
+        let isEnabled: Bool
+        let style: Stepper.Style
+        let action: () -> Void
+        
+        var body: some View {
+            SwiftUI.Button(action: action) {
+                icon
+                    .background(
+                        (isEnabled ? style.backgroundActiveColor : style.backgroundColor)
+                            .clipShape(shape)
+                    )
+            }
+            .disabled(!isEnabled)
+        }
+        
+        var icon: Icon {
+            let color = isEnabled ? style.foregroundActiveColor : style.foregroundColor
+
+            return isIncrement ? .init(.plus, color: color) : .init(.minus, color: color)
+        }
+        
+        var shape: some InsettableShape {
+            Capsule()
+        }
+    }
+}
+
+// MARK: - Styles
+extension Stepper {
+    
+    public enum Style {
+        case primary
+        case secondary
+        
+        public var foregroundColor: Color {
+            switch self {
+            case .primary:                  return .white.opacity(0.5)
+            case .secondary:                return .inkDark.opacity(0.5)
+            }
+        }
+        
+        public var foregroundActiveColor: Color {
+            switch self {
+            case .primary:                  return .white
+            case .secondary:                return .inkDark
+            }
+        }
+        
+        public var backgroundColor: Color {
+            switch self {
+            case .primary:                  return .blueNormal.opacity(0.5)
+            case .secondary:                return .cloudNormal.opacity(0.5)
+            }
+        }
+        
+        public var backgroundActiveColor: Color {
+            switch self {
+            case .primary:                  return .blueNormal
+            case .secondary:                return .cloudNormal
+            }
+        }
+    }
+}
+
+// MARK: - Identifiers
+public extension AccessibilityID {
+    
+    static let stepperActionIncrement         = Self(rawValue: "orbit.stepper.increment")
```

Comment:
There is no reason for names to be different. Should be `let stepperIncrement` as in other components (removing only the `orbit` prefix).

Reply:
Renamed